lambda/jobresultprocessor/lambda_function.py

```diff
@@ -116,13 +116,18 @@ def processRequest(request):
     print("path: " +  path)
     maxPages = 100
     comprehendClient = ComprehendHelper()
-    processedComprehendData = comprehendClient.processComprehend(outputBucketName, 'response.json', path, maxPages)
+    comprehendandMedicalEntities = comprehendClient.processComprehend(outputBucketName, 'response.json', path, maxPages)
 
     print("DocumentId: {}".format(jobTag))
-    print("Processed Comprehend Data: {}".format(processedComprehendData))
-
-    # index document once the comprehend entities have been extracted
-    opg.indexDocument(docText,processedComprehendData)
+    print("Processed Comprehend Data: {}".format(comprehendandMedicalEntities))
+
+    # index document once the comprehend entities and KVPairs have been extracted
+    for key, val in opg_output["KVPairs"].items():
```

Comment:
In cases where we have String for values (for example the key: KVPairs), can we move these to constants?

Reply:
Sure. I will do the changes.